packages/core/types/src/http/product/common.ts

```diff
@@ -0,0 +1,192 @@
+import { BaseFilterable, OperatorMap } from "../../dal"
+import { BaseProductCollection } from "../collection/common"
+
+export type ProductStatus = "draft" | "proposed" | "published" | "rejected"
+
+export interface BaseProduct {
```

Comment:
Q: those the base interface takes into concideration the not allowed fields from the store end point? just to not leak something that is not allowed but is present in the types

Reply:
@adrien2p the `Base<Entity>` typings will only have the types that are shared across both, and then each of `admin` and `store` will additionally extend what is specific to them. Does that make sense?

Reply:
Yes that make sense, we will have to identify what is not allowed in the store 👍